src/gui/qgsexpressionstoredialog.h

```diff
@@ -57,9 +57,15 @@ class GUI_EXPORT QgsExpressionStoreDialog : public QDialog, private Ui::QgsExpre
      */
     QString helpText() const;
 
+    /**
+     * Returns whether the label text was modified
+     */
+    bool isLabelModified() { return mLabel->text() != mOriginalLabel; }
```

Comment:
Can you skip this one from the sip bindings? It's an internal detail only, shouldn't be part of stable api